<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CptDlg</class>
 <widget class="QDialog" name="CptDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBoxGray">
   <property name="geometry">
    <rect>
     <x>668</x>
     <y>207</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>灰度图像</string>
   </property>
   <widget class="QCheckBox" name="checkBoxGray">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>灰度图像</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxImageType">
   <property name="geometry">
    <rect>
     <x>666</x>
     <y>46</y>
     <width>341</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>图像源</string>
   </property>
   <widget class="QRadioButton" name="radioButtonDDR">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>96</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DDR</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonCmos">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>55</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CMOS</string>
    </property>
   </widget>
  </widget>
  <widget class="CptPreLabel" name="labelPic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>710</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>710</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxAdjustMent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>535</y>
     <width>1001</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>坐标微调</string>
   </property>
   <widget class="QPushButton" name="btnSkeletonLeft">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>左</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSkeletonRight">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>右</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSkeletonUp">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>25</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>上</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSkeletonDown">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>112</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>下</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBottomRight">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>68</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(200,200)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonLeftTop">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>49</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>125</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>左顶点</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLeftTop">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>50</y>
      <width>68</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(100,100)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonBottomRight">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>109</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>125</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>右下点</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxWay">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>319</y>
     <width>341</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>灰度通道</string>
   </property>
   <widget class="QRadioButton" name="radioButtonBrightWay">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>29</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>亮度通道</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonRWay">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>R通道</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonBWay">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>151</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>B通道</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonGWay">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>112</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>G通道</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CptPreLabel</class>
   <extends>QLabel</extends>
   <header>ImgpreprocessModule/Roi/CptPreLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
